test/java/build.gradle

```diff
@@ -0,0 +1,44 @@
+apply plugin: 'java-library'
+
+repositories {
+    mavenLocal()
+    jcenter()
+}
+
+task generateSysl(type:Exec) {
+  commandLine 'reljam',  '--out', 'src/main/java', 'model', '/tuplecomplex', 'UserFormComplex'
+
+  standardOutput = new ByteArrayOutputStream()
+
+  ext.output = {
+    return standardOutput.toString()
+  }
+}
+
+compileJava.dependsOn generateSysl
+
+sourceSets {
+    main {
+        java {
+            srcDirs = ["src/main/java", "../../src/libs/java"]
+        }
+        resources {
+            srcDir 'src/main/resources'
```

Comment:
Move resources to top level please and change this line to:
`srcDir "resources"`

Reply:
I thought src/main/{java,resources} was the standard java structure.
https://maven.apache.org/guides/introduction/introduction-to-the-standard-directory-layout.html

Reply:
Hm, but it's `tests` we have got here not `main` code, so it should at least be `src/test/{java,resources} `
It seems like the right way at the `repo_root` but 
`tests/java/src/java` or `tests/java/test/java` reads strange.
On that note, the right solution might be to put the `build.gradle` file in the `repo_root` directory.
This would also make the CI call cleaner:
`gradle test` rather than `cd tests/java && gradle test`